### PR TITLE
Added Contributing guidelines and updated readme

### LINKED ISSUE #3: [FEATURE]  Adding a CONTRIBUTING.md file
### Is this a unique feature?

- [X] I have checked "open" AND "closed" issues and this is not a duplicate

### Is your feature request related to a problem/unavailable functionality? Please describe.

The Repo lacks a CONTRIBUTING.md file.

### Proposed Solution

Adding a valid CONTRIBUTING.md file to make it easier for other folks to Contribute.

### Screenshots

_No response_

### Do you want to work on this issue?

No

### If you want to work on this issue...

_No response_

### PR DESCRIPTION
**Pull request by IrfanSalim**
Closes #3

### CODE REVIEW

**Comment by guranshdeol** [comment]
Link to Contribution.md is of the file present in fork, not of the main project repo.

File: Readme.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ npm start
 
 Contributions are always welcome! If you'd like to contribute to this project please fork the repository and create a pull request.
 
-  
+Check out <a href="https://github.com/srummanf/Invoice-Generator/blob/master/CONTRIBUTING.md">CONTRIBUTION</a> guidelines for more details.
```
Comment (guranshdeol) [inline Readme.md]: Here the link to contribution.md should be of the main repo not the fork.

**Review by guranshdeol — CHANGES_REQUESTED** [review]

File: Readme.md (hunk 1 above)
Comment (IrfanSalim) [reply Readme.md]: Sorry about the miss, this is fixed now.

**Comment by guranshdeol** [comment]
👍 looks good
consider starring this repositry.

**Pull request merged**